cpp/src/ray/config_internal.cc

```diff
@@ -220,6 +220,9 @@ void ConfigInternal::Init(RayConfig &config, int argc, char **argv) {
 };
 
 void ConfigInternal::SetBootstrapAddress(std::string_view address) {
+  if (address == "auto") {
+    return;
+  }
```

Comment:
It seems this isn't the right approach. We just want to connect to the existing Ray cluster here; we've set `RAY_ADDRESS="auto"` for the JobSupervisorActor.

Reply:
@architkulkarni Sorry I miss this case, I've been a little busy lately, I'll fix it up tomorrow

Reply:
Ah no problem, actually this part was just an attempted change by me to see if it would work, but my guess was wrong.  I've messaged you offline with more details